plugin.video.bcf/resources/lib/plugin.py

```diff
@@ -0,0 +1,260 @@
+# -*- coding: utf-8 -*-
+"""Main plugin file - Handles the various routes"""
+
+__author__ = "fraser"
+
+import routing
+import xbmc
+import xbmcaddon as xa
+import xbmcplugin as xp
+from xbmcgui import ListItem
+
+from resources.lib import kodiutils as ku
+from resources.lib import search as bcfs
+
+plugin = routing.Plugin()
+ADDON = xa.Addon()
+ADDON_ID = ADDON.getAddonInfo("id")  # plugin.video.bcf
+ADDON_NAME = ADDON.getAddonInfo("name")  # British Council Film Player
+MEDIA_URI = "special://home/addons/{}/resources/media/".format(ADDON_ID)
+
+BCF_THEMES = [
+    "Any",
```

Comment:
would be nice to see these as localized strings in the future if someone wants to provide translations to your addon via transifex